lib/workers/repository/onboarding/pr/index.ts

```diff
@@ -126,6 +130,13 @@ If you need any further assistance then you can also [request help here](${
   if (is.string(config.prFooter)) {
     prBody = `${prBody}\n---\n\n${template.compile(config.prFooter, config)}\n`;
   }
+
+  const configFile = defaultConfigFile(config);
+  const existingContents =
+    (await getFile(configFile, config.onboardingBranch)) ?? '';
+  const hash = toSha256(existingContents);
+  prBody += `\n<!--renovate-config-hash:${toBase64(hash)}-->`;
```

Comment:
i don't think we need to encode the Sha to base64, ad it's already a base64 string.
```suggestion
  prBody += `\n<!--renovate-config-hash:${hash}-->`;
```